sbin/cubeit

```diff
@@ -257,8 +257,20 @@ fi
 
 ima_sign()
 {
+
+    # evmctl has problem with parsing a relative path of IMA key
+    # when recusively signing files under a directory
+    # so we convert KEYS_DIR into abs path
+    if test ! $(which realpath); then
```

Comment:
rather than test, just use [ ] to be consistent with the (most) of the code

Reply:
thanks, I will modify the patch and resend v2 soon.
Close this PR.